pkg/goDB/conditions/tokenize.go

```diff
@@ -70,26 +69,19 @@ func SanitizeUserInput(conditional string) (string, error) {
 	}
 
 	// first, convert everything to lower case
-	r, err = regexp.Compile(".*")
-	if err != nil {
-		return sanitized, err
-	}
+	r = regexp.MustCompile(".*")
 
 	sanitized = string(r.ReplaceAllFunc([]byte(conditional), bytes.ToLower))
 
 	// range over map to convert the individual entries
 	for condGrammarOp, userGrammarOps := range grammarConversionMap {
 		for _, userOp := range userGrammarOps {
-			r, err = regexp.Compile(userOp)
-			if err != nil {
-				return sanitized, err
-			}
-
+			r = regexp.MustCompile(userOp)
```

Comment:
If the `grammarConversionMap` is static, compiling the regexes _once_ would make sense as well IMHO.

Reply:
Absolutely.